Graphics/GraphicsEngineVulkan/include/VulkanUtilities/VulkanCommandBuffer.hpp

```diff
@@ -138,6 +138,32 @@ class VulkanCommandBuffer
         vkCmdDrawIndexedIndirect(m_VkCmdBuffer, Buffer, Offset, DrawCount, Stride);
     }
 
+    __forceinline void DrawMesh(uint32_t TaskCount, uint32_t FirstTask)
+    {
+#ifdef VK_NV_mesh_shader
+        VERIFY_EXPR(m_VkCmdBuffer != VK_NULL_HANDLE);
+        VERIFY(m_State.RenderPass != VK_NULL_HANDLE, "vkCmdDrawMeshTasksNV() must be called inside render pass (19.3)");
+        VERIFY(m_State.GraphicsPipeline != VK_NULL_HANDLE, "No graphics pipeline bound");
+
+        vkCmdDrawMeshTasksNV(m_VkCmdBuffer, TaskCount, FirstTask);
+#else
+        UNSUPPORTED("DrawMesh is not supported in current Vulkan headers");
+#endif
+    }
+
+    __forceinline void DrawMeshIndirect(VkBuffer Buffer, VkDeviceSize Offset, uint32_t DrawCount, uint32_t Stride)
+    {
+#ifdef VK_NV_mesh_shader
```

Comment:
This is not really required as our headers do have mesh shader

Reply:
vkCmdDrawMeshTasksIndirectNV doesn't exists if we statically linking with vulkan

Reply:
can be replaced by `#if DILIGENT_USE_VOLK`

Reply:
Ah, this is required to make it compile on Android, I see.